challenge-15/challenge-15.js

```diff
@@ -1,54 +1,81 @@
-/*
-Envolva todo o código desse desafio em uma IIFE.
-Crie um arquivo chamado index.html e adicione esse script ao HTML.
-*/
+(function(){
+    /*
+    Envolva todo o código desse desafio em uma IIFE.
+    Crie um arquivo chamado index.html e adicione esse script ao HTML.
+    */
 
-/*
-Crie uma função construtora chamada "Person". Esse construtor deve ter
-as seguintes características:
-- Deve receber 3 parâmetros: `name`, `lastName` e `age`;
-- Deverá ter 3 propriedades:
-  - `name` - que receberá o valor do parâmetro `name`;
-  - `lastName` - que receberá o valor do parâmetro `lastName`;
-  - `age` - que receberá o valor do parâmetro `age`;
-- Deverá ter 3 métodos:
-  - `getFullName` - que deve retornar o nome completo do objeto criado,
-  no formato:
-    - "[NAME] [LASTNAME]"
-  - `getAge` - que deverá retornar a idade (age);
-  - `addAge` - esse método não deverá ter nenhum parâmetro, mas ao invocá-lo
-  iremos passar um único argumento, que é a quantidade de anos que devem ser
-  adicionados à idade original (age). Esse método deverá retornar o objeto
-  que será instanciado.
-*/
-// ?
+    /*
+    Crie uma função construtora chamada "Person". Esse construtor deve ter
+    as seguintes características:
+    - Deve receber 3 parâmetros: `name`, `lastName` e `age`;
+    - Deverá ter 3 propriedades:
+      - `name` - que receberá o valor do parâmetro `name`;
+      - `lastName` - que receberá o valor do parâmetro `lastName`;
+      - `age` - que receberá o valor do parâmetro `age`;
+    - Deverá ter 3 métodos:
+      - `getFullName` - que deve retornar o nome completo do objeto criado,
+      no formato:
+        - "[NAME] [LASTNAME]"
+      - `getAge` - que deverá retornar a idade (age);
+      - `addAge` - esse método não deverá ter nenhum parâmetro, mas ao invocá-lo
+      iremos passar um único argumento, que é a quantidade de anos que devem ser
+      adicionados à idade original (age). Esse método deverá retornar o objeto
+      que será instanciado.
+    */
+    function Person( name, lastName, age ){
+        this.name = name;
+        this.lastName = lastName;
+        this.age = age;
+        this.getFullName = function(){
+            return name + ' ' + lastName;
+        }
+        this.getAge = function(){
+            return age;
+        }
+        this.addAge = function(){
+            age = age + arguments[0];
```

Comment:
Porque aqui eu não precisei passar o <code>this</code> para o <code>age</code> ?